### PR TITLE
ISS-73 # Load - Junit Load Runner

### PR DESCRIPTION
**Pull request by nirmalchandra**

### CODE REVIEW

File: src/main/java/org/jsmart/zerocode/core/runner/parallel/ZeroCodeLoadRunner.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    protected Description describeChild(TestMapping child) {
+        this.testDescription = createTestDescription(testClass, LOAD_LABEL + child.testMethod());
+        return testDescription;
+    }
+
+    @Override
+    protected void runChild(TestMapping child, RunNotifier notifier) {
+        notifier.fireTestStarted(testDescription);
+
+        boolean failed = loadProcessor
```
Comment (authorjapps) [inline src/main/java/org/jsmart/zerocode/core/runner/parallel/ZeroCodeLoadRunner.java]: can you rename to `didFail` or `isFailed`

File: src/main/java/org/jsmart/zerocode/core/runner/parallel/ZeroCodeMultiLoadRunner.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+import org.junit.runner.notification.RunNotifier;
+import org.junit.runners.ParentRunner;
+import org.junit.runners.model.InitializationError;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class ZeroCodeMultiLoadRunner extends ParentRunner<TestMapping> {
```
Comment (authorjapps) [inline src/main/java/org/jsmart/zerocode/core/runner/parallel/ZeroCodeMultiLoadRunner.java]: No usage?

File: src/main/java/org/jsmart/zerocode/core/runner/parallel/ZeroCodeLoadRunner.java [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+import org.junit.runners.model.InitializationError;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.runner.Description.createTestDescription;
+
+public class ZeroCodeLoadRunner extends ParentRunner<TestMapping> {
```
Comment (siddhagalaxy) [inline src/main/java/org/jsmart/zerocode/core/runner/parallel/ZeroCodeLoadRunner.java]: Can you cover this with junit?

File: src/test/resources/load_config_test.properties [hunk 4]
```diff
@@ -1,23 +1,25 @@
-# You can enter as many threads to stimulate a load test. A single user is represented by each Thread.
-# So if you wish to simulate a load test with 5 concurrent users then you need to enter 5 as the value
-# for this property.
-number.of.threads=3
+# You can enter as many threads to stimulate a load test. A single user is represented by each Thread. So if you wish
```
Comment (siddhagalaxy) [inline src/test/resources/load_config_test.properties]: These should go into a Wiki page or README please.

File: src/main/java/org/jsmart/zerocode/core/runner/parallel/ZeroCodeLoadRunner.java (hunk 3 above)
Comment (nirmalchandra) [reply src/main/java/org/jsmart/zerocode/core/runner/parallel/ZeroCodeLoadRunner.java]: yes, done

File: src/main/java/org/jsmart/zerocode/core/runner/parallel/ZeroCodeLoadRunner.java (hunk 1 above)
Comment (nirmalchandra) [reply src/main/java/org/jsmart/zerocode/core/runner/parallel/ZeroCodeLoadRunner.java]: changed to `hasFailed`, sounds better.

File: src/main/java/org/jsmart/zerocode/core/runner/parallel/ZeroCodeMultiLoadRunner.java (hunk 2 above)
Comment (nirmalchandra) [reply src/main/java/org/jsmart/zerocode/core/runner/parallel/ZeroCodeMultiLoadRunner.java]: still WIP-, but the usage is not yet defined,will raise an issue and add to tech-debt or backlog/enhancement if needed.

**Pull request merged**